diff --git a/node_modules/react-native-animated-ellipsis/dist/AnimatedEllipsis.js b/node_modules/react-native-animated-ellipsis/dist/AnimatedEllipsis.js
index 908626b..2123d8c 100644
--- a/node_modules/react-native-animated-ellipsis/dist/AnimatedEllipsis.js
+++ b/node_modules/react-native-animated-ellipsis/dist/AnimatedEllipsis.js
@@ -1,5 +1,5 @@
 import React, { Component } from 'react';
-import { Text, Animated, View, StyleSheet } from 'react-native';
+import { Text, Animated, View, StyleSheet, TextStyle } from 'react-native';
 import PropTypes from 'prop-types';
 
 export default class AnimatedEllipsis extends Component {
@@ -47,7 +47,8 @@ export default class AnimatedEllipsis extends Component {
 
     Animated.timing(this._animation_state.dot_opacities[which_dot], {
       toValue: this._animation_state.target_opacity,
-      duration: this.props.animationDelay
+      duration: this.props.animationDelay,
+      useNativeDriver: true
     }).start(this.animate_dots.bind(this, next_dot));
   }
 
@@ -71,7 +72,7 @@ AnimatedEllipsis.propTypes = {
   numberOfDots: PropTypes.number,
   animationDelay: PropTypes.number,
   minOpacity: PropTypes.number,
-  style: Text.propTypes.style
+  style: PropTypes.shape(TextStyle)
 };
 AnimatedEllipsis.defaultProps = {
   numberOfDots: 3,
